packages/legend-art/style/base/_panel.scss

```diff
@@ -38,6 +38,7 @@
   &__header__actions {
     display: flex;
     height: 100%;
+    align-items: center;
```

Comment:
why do we do this? isn't `header__actions` have height `2.8rem` anyway so this will be obvious that `header__action` will fill the height so this is not needed?